src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml

```diff
@@ -1,13 +1,13 @@
 <zapaddon>
 	<name>Passive scanner rules (alpha)</name>
-	<version>10</version>
+	<version>11</version>
```

Comment:
Ah, I didn't know that. I thought every new version is automatically pushed to the marketplace. What is the process to release to market place? And who does that?